moto/utilities/docker_utilities.py

```diff
@@ -31,3 +31,17 @@ def replace_adapter_send(*args, **kwargs):
 
                 self.docker_client.api.get_adapter = replace_adapter_send
         return self.__docker_client
+
+
+def parse_image_name(image_name):
+    # podman does not support short container image name out of box - try to make a full name
+    if ":" in image_name:
+        image_repository, image_tag = image_name.split(":", 1)
+    else:
+        image_repository = image_name
+        image_tag = "latest"
+    if "/" not in image_repository:
+        image_repository = "library/" + image_repository
+    if len(image_repository.split("/")) < 3:
+        image_repository = "docker.io/" + image_repository
```

Comment:
Can you make the `docker.io` configurable, just in case people need to use a different repository? Ideally it's an env-variable in settings.py

Reply:
Sure! I've also found a bug in this function. Will update both soon.

Reply:
Done! I added `$DEFAULT_CONTAINER_REGISTRY`. Please tell me if you want a better name.